File: Iniciante/URI 1847.cpp
// https://www.beecrowd.com.br/judge/pt/problems/view/1847

#include <iostream>
using namespace std;
int main() 
{
    int a, b, c; cin >> a >> b >> c;
    if (a > b && b <= c)
    cout << ":)" << endl;
    else if (a < b && b >= c)
    cout << ":(" << endl;
    else if (a < b && b < c && (b-c) > (a-b))
    cout << ":(" << endl;
    else if (a < b && b < c && (b-c) <= (a-b))
    cout << ":)" << endl;
    else if (a > b && b > c && (b-c) < (a-b))
    cout << ":)" << endl;
    else if (a > b && b > c && (b-c) >= (a-b))
    cout << ":(" << endl;
    else if (a == b && b < c)
    cout << ":)" << endl;
    else if (a == b && b > c)
    cout << ":(" << endl;
    else
    cout << ":(" << endl;
}

File: Iniciante/URI 1847.py
# https://www.beecrowd.com.br/judge/pt/problems/view/1847

a, b, c= map(int, input().split())
if a > b and b <= c:
    print(":)")
elif a < b and b >= c:
    print(":(")
elif a < b and b < c and (c-b) < (b-a):
    print(":(")
elif a < b and b < c and (c-b) >= (b-a):
    print(":)")
elif a > b and b > c and (c-b) > (b-a):
    print(":)")
elif a > b and b > c and (c-b) <= (b-a):
    print(":(")
elif a == b and b < c:
    print(":)")
elif a == b and b > c:
    print(":(")
else:
    print(":(")